src/stores/lava-simulation-store.ts

```diff
@@ -99,10 +101,6 @@ export const LavaSimulationStore = types
         ventLongitude: self.ventLongitude
       };
       self.worker.postMessage({ type: "start", parameters });
-
-      return () => {
-        self.worker?.terminate();
-      };
```

Comment:
Seems like we would still want the worker to be terminated when the component is unmounted, so I would think we still want to return this termination function from the `useEffect`.

Reply:
This is actually not in a `useEffect`, I think it was at some point during development, which is why this return statement was added. But it hasn't been doing anything for a while now.